python/metatensor-torch/metatensor/torch/atomistic/openmm_force.py

```diff
@@ -0,0 +1,286 @@
+import warnings
+from typing import Iterable, List, Optional
+
+import torch
+
+from metatensor.torch import Labels, TensorBlock
+from metatensor.torch.atomistic import (
+    ModelEvaluationOptions,
+    ModelOutput,
+    NeighborListOptions,
+    System,
+    load_atomistic_model,
+)
+
+from .ase_calculator import STR_TO_DTYPE
+
+
+try:
+    import NNPOps.neighbors
+    import openmm
+    import openmmtorch
+
+    HAS_OPENMM = True
+except ImportError:
+
+    class openmm:
+        class System:
+            pass
+
+        class Force:
+            pass
+
+        class app:
+            class Topology:
+                pass
+
+    HAS_OPENMM = False
+
+
+def get_metatensor_force(
+    system: openmm.System,
+    topology: openmm.app.Topology,
+    path: str,
+    extensions_directory: Optional[str] = None,
+    forceGroup: int = 0,
+    selected_atoms: Optional[Iterable[int]] = None,
+    check_consistency: bool = False,
+) -> openmm.Force:
+    """
+    Create an OpenMM Force from a metatensor atomistic model.
+
+    :param system: The OpenMM System object.
+    :param topology: The OpenMM Topology object.
+    :param path: The path to the exported metatensor model.
+    :param extensions_directory: The path to the extensions for the model.
+    :param forceGroup: The force group to which the force should be added.
```

Comment:
should we link to OpenMM docs for force group here? It might not be clear to everyone.